engine/tests/ElggCoreGetEntitiesFromAnnotationsTest.php

```diff
@@ -330,13 +330,17 @@ public function testElggGetEntitiesFromAnnotationsCalculateConstrainedByWhere()
 			}
 		}
 
+		$prefix = _elgg_services()->db->getTablePrefix();
 		$options = array(
 			'type' => 'object',
 			'subtypes' => $subtypes,
 			'limit' => $num_entities,
 			'annotation_name' => $name,
 			'annotation_values' => array_unique(call_user_func_array('array_merge', $values)),
 			'calculation' => 'sum',
+			'joins' => array(
+				"JOIN {$prefix}metastrings msv ON (n_table.value_id = msv.id)",
```

Comment:
I added manual re-ordering to _elgg_entities_get_metastrings_options() to fix this.